website/data/docs-nav-data.json

```diff
@@ -43,6 +43,10 @@
       {
         "title": "Using GitHub Actions to Unlock GitOps Workflows",
         "path": "use-cases/waypoint-gh-actions"
+      },
+      {
+        "title": "Service Mesh Microservice Deployment",
+        "path": "intro/use-cases/service-mesh-microservice-deployment"
```

Comment:
should remove `intro/` because i moved the nav up a level

Reply:
Fixed!